pyreisejl/utility/const.py

```diff
@@ -1,9 +1,13 @@
-import posixpath
+import os
+from pathlib import Path
 
-DATA_ROOT_DIR = "/mnt/bes/pcm"
+if os.getenv("DEPLOYMENT_MODE") is not None:
+    DATA_ROOT_DIR = os.path.join(Path.home(), "ScenarioData", "")
+else:
+    DATA_ROOT_DIR = "/mnt/bes/pcm"
 
-SCENARIO_LIST = posixpath.join(DATA_ROOT_DIR, "ScenarioList.csv")
-EXECUTE_LIST = posixpath.join(DATA_ROOT_DIR, "ExecuteList.csv")
-EXECUTE_DIR = posixpath.join(DATA_ROOT_DIR, "tmp")
-INPUT_DIR = posixpath.join(DATA_ROOT_DIR, "data", "input")
-OUTPUT_DIR = posixpath.join(DATA_ROOT_DIR, "data", "output")
+SCENARIO_LIST = os.path.join(DATA_ROOT_DIR, "ScenarioList.csv")
+EXECUTE_LIST = os.path.join(DATA_ROOT_DIR, "ExecuteList.csv")
+EXECUTE_DIR = os.path.join(DATA_ROOT_DIR, "tmp")
+INPUT_DIR = os.path.join(DATA_ROOT_DIR, "data", "input")
+OUTPUT_DIR = os.path.join(DATA_ROOT_DIR, "data", "output")
```

Comment:
One day, it will be nice to find a ways to not set these variables when we use **PowerSimData** to launch **REISE.jl**. What is particularly ugly is that the path needs to be identical to the ones set in **PowerSimData**